### PR TITLE
Fix combining crystallographic protein structure and docked ligand SDF file

### PR DESCRIPTION
**Pull request by kaminow**
## Description
Fix the script for creating a docked pose from the protein crystal structure and docked ligand SDF file. Add the function for doing this combination to `asapdiscovery.data.openeye`.

## Status
- [x] Ready to go

### CODE REVIEW

File: asapdiscovery-docking/asapdiscovery/docking/scripts/make_docked_complexes.py [hunk 1]
```diff
@@ -3,19 +3,17 @@
...
 import pandas
-from openeye import oechem
-
-sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 from asapdiscovery.data.openeye import load_openeye_pdb  # noqa: E402
 from asapdiscovery.data.openeye import load_openeye_sdf  # noqa: 402
 from asapdiscovery.data.openeye import save_openeye_pdb  # noqa: 402
 from asapdiscovery.data.openeye import split_openeye_mol  # noqa: E402
+from asapdiscovery.data.openeye import combine_protein_ligand
+from openeye import oechem
```
Comment (hmacdope) [inline asapdiscovery-docking/asapdiscovery/docking/scripts/make_docked_complexes.py:13]: now that #161 is merged you should import `oechem` from `asapdiscovery.data.openeye`

File: asapdiscovery-data/asapdiscovery/data/openeye.py [hunk 2]
```diff
@@ -5,6 +5,78 @@
...
+
+    Returns
+    -------
+    oechem.OEMol
+        Combined molecule, with the appropriate biopolymer field set for the lig atoms
+    """
+    # Calculate residue number if necessary
+    if resid is None:
+        # Find max resid for numbering the ligand residue
+        resid = max([r.GetResidueNumber() for r in oechem.OEGetResidues(prot)]) + 1
```
Comment (hmacdope) [inline asapdiscovery-data/asapdiscovery/data/openeye.py:37]: would probably be good to document why all these +1s are necessary in the docstring. I assume it is a 0 vs 1 based indexing thing, but good for someone following later.

**Review by hmacdope — CHANGES_REQUESTED** [review]
Nitpick + quick docs thing :)

File: asapdiscovery-docking/asapdiscovery/docking/scripts/make_docked_complexes.py (hunk 1 above)
Comment (kaminow) [reply asapdiscovery-docking/asapdiscovery/docking/scripts/make_docked_complexes.py]: should be fixed in [eb14cdc](https://github.com/choderalab/asapdiscovery/pull/160/commits/eb14cdcac7667f385ed81ef100e07b4a0735015f)

File: asapdiscovery-data/asapdiscovery/data/openeye.py (hunk 2 above)
Comment (kaminow) [reply asapdiscovery-data/asapdiscovery/data/openeye.py:38]: added in [c1cf66f](https://github.com/choderalab/asapdiscovery/pull/160/commits/c1cf66f8eec4eda12987b36a387ff10bbbb02dfd)

**Review by hmacdope — APPROVED** [review]
LGTM!

**Pull request merged**